### PR TITLE
grub2 install: Support secure boot on powerpc (bsc#1192764 jsc#SLE-18271).

### PR DESCRIPTION
**Pull request by hramrach**

### CODE REVIEW

**Comment by hramrach** [comment]
Seems to work as designed:

(1/1) Installing: grub2-powerpc-ieee1275-2.06-31.1.noarch ...........................................................................[done]
Executing %posttrans script 'grub2-powerpc-ieee1275-2.06-31.1.noarch.rpm' --------------------------------------------------------------[/]2022-01-07 15:20:10 <1> update-bootloader-5650 new.126: update-bootloader-5650 = /sbin/update-bootloader, version = 0.936, root = 0:47
2022-01-07 15:20:10 <1> update-bootloader-5650 main.269: /sbin/update-bootloader --reinit
2022-01-07 15:20:10 <1> update-bootloader-5650 main.271: bootloader = grub2
2022-01-07 15:20:10 <1> update-bootloader-5650 main.277: locale = 
2022-01-07 15:20:14 <1> update-bootloader-5650 run_command.294: '/usr/lib/bootloader/grub2/install' = 0, output:
<<<<<<<<<<<<<<<<
target = powerpc-ieee1275
+ dd status=none if=/usr/share/grub2/powerpc-ieee1275/grub.elf of=/dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_scsi0-hd0-part1
>>>>>>>>>>>>>>>>
2022-01-07 15:22:09 <1> update-bootloader-5650 run_command.294: '/usr/lib/bootloader/grub2/config' = 0, output:
<<<<<<<<<<<<<<<<
+ /usr/sbin/grub2-mkconfig -o /boot/grub2/grub.cfg
Generating grub configuration file ...
Found linux image: /boot/vmlinux-5.14.21-19.g90531c1-default
Found initrd image: /boot/initrd-5.14.21-19.g90531c1-default
Found linux image: /boot/vmlinux-5.14.21-18.gd34e66a-default
Found initrd image: /boot/initrd-5.14.21-18.gd34e66a-default
Found linux image: /boot/vmlinux-5.14.21-17.gd20e650-default.unsigned
Warning: os-prober will not be executed to detect other bootable partitions.
Systems on them will not be added to the GRUB boot configuration.
Check GRUB_DISABLE_OS_PROBER documentation entry.
done
>>>>>>>>>>>>>>>>
Executing %posttrans scripts ........................................................................................................[done]

File: grub2/install [hunk 1]
```diff
@@ -147,6 +147,14 @@ if [ "$SYS__BOOTLOADER__TRUSTED_BOOT" = yes -a -d "/usr/lib/trustedgrub2/$target
   trusted="--directory=/usr/lib/trustedgrub2/$target"
 fi
 
+if [ "$target" == "powerpc-ieee1275" ] ; then
```
Comment (wfeldt) [inline grub2/install:150]: Please use `=` for consistency here.

**Comment by wfeldt** [comment]
This is for both Factory and SLE15-SP4?

File: grub2/install (hunk 1 above)
Comment (hramrach) [reply grub2/install]: yes, it should be =, some versions of test complain about ==

**Comment by hramrach** [comment]
Yes, this is for Factory and 15 SP4

We might need something different in SP5 but we will only know when we get there.

**Comment by hramrach** [comment]
Technically you can merge this indiscriminately because the parameter is ignored with grub versions that don't implement the feature.

**Pull request merged**